core/test-framework.cabal

```diff
@@ -9,8 +9,8 @@ Description:         Allows tests such as QuickCheck properties and HUnit test c
 License:             BSD3
 License-File:        LICENSE
 Author:              Max Bolingbroke <batterseapower@hotmail.com>
-Maintainer:          Libraries List <libraries@haskell.org>
-Homepage:            http://haskell.github.io/test-framework/
+Maintainer:          Andreas Abel
+Homepage:            https://github.com/haskell/test-framework
```

Comment:
I think we want to link directly to the README.  I'm gonna open PR against your branch for that.